api-scripts/create-or-update-dataset.py

```diff
@@ -601,5 +599,6 @@ def _ui(args=None):
 
 if __name__ == '__main__':
     host, apikey, gmm_path = _ui()
-    main(host, apikey, gmm_path,
-         verify_ssl=(host.split('https://'[1].startswith('localhost'))))
+
+    is_localhost = not host.split('https://')[1].startswith('localhost')
```

Comment:
The `not` here and in `verify_ssl=(not is_localhost)` is putting us back where we started, I think! 
```suggestion
    is_localhost = host.split('https://')[1].startswith('localhost')
```

Reply:
+ 1 on this - otherwise it looks good to me!

Reply:
Ugh, sorry about the sloppiness here!  And thank you both for your patience

Reply:
No worries at all; I have 100% been there. Somehow it always seems to be the tiny changes where silly mistakes happen most easily!